.eslintrc.js

```diff
@@ -1,16 +1,30 @@
 module.exports = {
   root: true,
-  extends: '@react-native-community',
+  extends: [
+    '@react-native-community',
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
+  plugins: ['@typescript-eslint', 'import', 'unused-imports'],
```

Comment:
could you explain `import` and `unused-imports` plugins?

Reply:
`import` and `unused-imports` are also config for the other 2 projects, they include rules:
- Warn if there are cycle import
- Error if import statements are not at the beginning of file
- Error if there are duplicate imports
- Error if there are unused imports